tests/test_check_runner.py

```diff
@@ -0,0 +1,145 @@
+"""Test suite for check_runner"""
+
+# pylint: disable=E1101, W0621, R0903
```

Comment:
Module-wide error suppressions are rarely a good thing. I'm fine with suppressing `R0903` on a per-class basis. `E1101` and `W0621` are really ought to be fixed though, or if not clearly suppressed inline.

Reply:
I knew you would've had something to say about this.